src/components/Video.tsx

```diff
@@ -90,12 +86,13 @@ function Video({goal}: {goal: Goal}) {
           >
             {buttonText}
           </button>
-          <button
```

Comment:
Same here, link instead of button

Reply:
Nice!! This fixes an annoying bug where it would glitch when opening reddit. 🙏